### ISSUE TITLE
Pftool fails to build as a subdirectory of marfs

### ISSUE THREAD

**Issue by Brofessional** [pftool/pftool#91]
I have a marfs fork with the dependencies required checked out as submodules so I can clone them all at once into a single directory. When trying to build as a subdirectory of marfs I get an error.

`$  ./autogen`
`libtoolize: putting macros in AC_CONFIG_MACRO_DIR, 'm4'.`
`libtoolize: copying file "m4/libtool.m4"`
`libtoolize: copying file "m4/ltoptions.m4"`
`libtoolize: copying file "m4/ltsugar.m4"`
`libtoolize: copying file "m4/ltversion.m4'`
`libtoolize: copying file "m4/lt~obsolete.m4"`
`libtoolize: Consider adding "-I m4" to ACLOCAL_AMFLAGS in Makefile.am.`
`configure.ac:98: error: required file "../ltmain.sh" not found`
`autoreconf: automake failed with exit status: 1`

Similarly trying to autoreconf

`$ autoreconf -i`
`libtoolize: Consider adding '-I m4' to ACLOCAL_AMFLAGS in Makefile.am.`
`configure.ac:98: error: required file '../ltmain.sh' not found`
`autoreconf: automake failed with exit status: 1`

```
$ ls -1 marfs
aclocal.m4
add_namespace
ar-lib
autom4te.cache
aws4c
common
compile
config.guess
config.h
config.h.in
config.log
config.status
config.sub
configure
configure.ac
COPYRIGHT.h
depcomp
Dockerfile
Documentation
erasureUtils
experimentation
fuse
fuse.log
install-sh
isa-l
libmarfs.a
LICENSE.txt
Makefile
Makefile.am
Makefile.in
marfs_config
marfs_fuse
marfs_fuse.pid
marfs_gc
marfs_packer
marfs_quota
marfs_repack
mc_path_convert
missing
PA2X
pftool
README.md
separate_log_threads_support
stamp-h1
test_statvfs
utilities
```

However when NOT sub directory the build goes as planned.

```
$ ls -1
marfs
pftool
$ cd pftool
$ ./autogen
libtoolize: putting auxiliary files in `.'.
libtoolize: copying file `./ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIR, `m4'.
libtoolize: copying file `m4/libtool.m4'
libtoolize: copying file `m4/ltoptions.m4'
libtoolize: copying file `m4/ltsugar.m4'
libtoolize: copying file `m4/ltversion.m4'
libtoolize: copying file `m4/lt~obsolete.m4'
libtoolize: Consider adding `-I m4' to ACLOCAL_AMFLAGS in Makefile.am.
```

**Comment by gransom** [comment]
The issue appears to be with the AC_PROG_LIBTOOL directive.  For some reason, existing as a subdir of marfs seems to be causing libtoolize to fail to create 'ltmain.sh'.
I'm still not sure why this is happening or why autotools seems to be looking in the parent directory for this script ('../ltmain.sh').  However, I did find that simply removing this directive both fixed the issue and did not seem to interfere with the build process (both with and without marfs enabled).  While I may investigate further as to why we see this odd behavior, my current plan is simply to remove the libtool directive.  The commit is sitting in my pftool fork.  I will be issuing a pull request for it next week.

**Comment by gransom** [comment]
Should be fixed through pull request #92 
Let me know if it's not!
